classes/ImageManager.php

```diff
@@ -86,7 +86,7 @@ public static function thumbnail($image, $cache_image, $size, $image_type = 'jpg
 		if (Context::getContext()->controller->controller_type == 'admin')
 			return '<img src="../img/tmp/'.$cache_image.($disable_cache ? '?time='.time() : '').'" alt="" class="imgm img-thumbnail" />';
 		else
-			return '<img src="'._PS_TMP_IMG_.$cache_image.($disable_cache ? '?time='.time() : '').'" alt="" class="imgm img-thumbnail" />';
+			return '<img src="'.Tools::getShopDomain(true)._PS_TMP_IMG_.$cache_image.($disable_cache ? '?time='.time() : '').'" alt="" class="imgm img-thumbnail" />';
```

Comment:
Hi,

Thank you for your contribution, what do you think about doing ```Context::getContext()->shop->getBaseURL(true, true)``` in order to have correct protocol ?

Best regards